### DISCUSSION TITLE
How to redirect to external page

### DISCUSSION THREAD

**Discussion by Andrew-web-coder** [vuejs/vitepress#2083] [Q&A] [answered]
Hi,

I would like to have a page, like `/smth/` that redirects to `example.com/else/`. If I try to use `useRouter`, then it redirects to just `/else`. So, how to do that?

**Comment by brc-dd** [comment, marked as answer]
You need to configure it in your hosting platform.

For Netlify: https://docs.netlify.com/routing/redirects/
For Vercel: https://vercel.com/docs/concepts/projects/project-configuration#redirects

**Reply by Andrew-web-coder** [reply]
OK, thanks. I am using "Cloudflare Pages" and I will try this - https://developers.cloudflare.com/pages/platform/redirects/

**Reply by brc-dd** [reply]
Just a remark, you need to keep `_redirects` in your public directory (`docs/public/_redirects` if using default settings).

**Reply by Andrew-web-coder** [reply]
Yes, I followed the docs and used Build command `npm run docs:build && cp _redirects docs/.vitepress/dist/_redirects` and it works perfectly. Thanks!

**Reply by brc-dd** [reply]
Yeah, I meant if you keep it in the public directory, VitePress will do the copying for you, so just `npm run docs:build` will work as the build command (no need of that extra `cp` thing).

**Reply by Andrew-web-coder** [reply]
Oh, wow, thanks for the explanation!
